lib/buffer.js

```diff
@@ -379,6 +378,16 @@ Buffer.prototype.toString = function(encoding, start, end) {
 };
 
 
+Buffer.prototype.toString = function() {
+  const length = this.length | 0;
```

Comment:
Neither, really.  I just like my variables to have the type I expect them to have.  I can remove it if you want.